src/features/expert-scoring/components/index.tsx

```diff
@@ -0,0 +1,287 @@
+        <DialogHeader>
+          <DialogTitle className="mb-5">Update Scores</DialogTitle>
+        </DialogHeader>
+        <Form {...form}>
+          <form
+            // eslint-disable-next-line @typescript-eslint/no-misused-promises
+            onSubmit={form.handleSubmit(onSubmit)}
+            className="space-y-2"
+          >
+            <div className="grid grid-cols-6 gap-2">
+              <span className="col-span-2">
+                <FormLabel>Name</FormLabel>
+              </span>
+              <span className="col-span-1">
+                <FormLabel>Score</FormLabel>
+              </span>
+              <span className="col-span-2">
+                <FormLabel>Comment (optional)</FormLabel>
+              </span>
+            </div>
+            {fields.map((field, index) => {
+              // name and value, value as input field
+              return (
+                <div className="grid grid-cols-6 gap-2" key={field.id}>
+                  <FormField
+                    control={form.control}
+                    name={`scores.${index}.name`}
+                    render={({ field }) => (
+                      <FormItem className="col-span-2">
+                        <FormControl>
+                          <AutoComplete
+                            {...field}
+                            options={[
+                              ...(expertScoreNamesInProject.data?.name.map(
+                                ({ value }) => ({ label: value, value: value }),
+                              ) ?? []),
+                            ]}
+                            placeholder="Score name"
+                            onValueChange={(value: Option) => {
+                              field.onChange(value.value);
+                            }}
+                            value={{ value: field.value, label: field.value }}
+                            disabled={false}
+                          />
+                        </FormControl>
+                        <FormMessage />
+                      </FormItem>
+                    )}
+                  />
+                  <FormField
+                    control={form.control}
+                    name={`scores.${index}.value`}
+                    render={({ field }) => (
+                      <FormItem>
+                        <FormControl>
+                          <Input {...field} type="number" step={0.01} />
+                        </FormControl>
+                        <FormMessage />
+                      </FormItem>
+                    )}
+                  />
+                  <FormField
+                    control={form.control}
+                    name={`scores.${index}.comment`}
+                    render={({ field }) => (
+                      <FormItem className="col-span-2">
+                        <FormControl>
+                          <Textarea
+                            {...field}
+                            placeholder="Comment"
```

Comment:
Do you think it would make sense to make this box larger? I think that users would add more than just a couple of words. Maybe we could shift the comment blow name + score and increase the height.